package.json

```diff
@@ -147,8 +147,8 @@
         "postinstall": "husky install",
         "prettier:write": "prettier --write \"src/{main/webapp,test}/**/*.{json,ts,js,css,scss,html}\"",
         "prettier:check": "prettier --check \"src/{main/webapp,test}/**/*.{json,ts,js,css,scss,html}\"",
-        "lint": "eslint . --ext .ts",
-        "lint:fix": "eslint . --ext .ts --fix",
+        "lint": "eslint src --ext .ts",
```

Comment:
Could you limit the search space even further with a pattern similar to the one used for prettier above?
Then it could be a little bit faster by not traversing the `java/` directories that do not contain any `ts` files anyway.